amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/dashboard/model/TableRuntimeBean.java

```diff
@@ -0,0 +1,350 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.amoro.server.dashboard.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class TableRuntimeBean implements Serializable {
```

Comment:
Could TableRuntimeMeta be used here?
see https://github.com/apache/amoro/blob/master/amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/table/TableRuntimeMeta.java

This class is meant to load from table_runtime

Reply:
`TableRuntimeMeta` contains too many fields that we don't need(or will retrieved from `TableRuntime`) so create a new bean class here. I can reuse the `TableRuntimeMeta` here if we prefer this way, what do you think about this?

Reply:
I think reusing is better.
We could raise a new select sql...